msteams-platform/sbs-gs-commandbot.yml

```diff
@@ -270,8 +280,9 @@ items:
        ``` bash
        teamsfx preview
        ```
-
     ---
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Deploy+your+app+to+Azure&&author=%40surbhigupta&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-gs-commandbot%3Ftabs%3Dvscode%26tutorial-step%3D3&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-gs-commandbot.yml&documentVersionIndependentId=e7ac1764-9172-9f3e-2091-5dfd755eded4&metadata=*%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A*%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
Let's add 2 buttons here:

Deploy your app to Azure using Visual Studio Code
Deploy your app to Azure using Command Line

Reply:
Updated as suggested